### PR TITLE
chore: remove pallet::getter usage

### PR DESCRIPTION
**Pull request by playX18**

### CODE REVIEW

File: pallets/gear-bank/src/lib.rs [hunk 1]
```diff
@@ -158,15 +158,18 @@ pub mod pallet {
...
     pub type Bank<T> = StorageMap<_, Identity, AccountIdOf<T>, BankAccount<BalanceOf<T>>>;
 
     // Private storage that keeps amount of value that wasn't sent because owner is inexistent account.
     #[pallet::storage]
-    #[pallet::getter(fn unused_value)]
     pub type UnusedValue<T> = StorageValue<_, BalanceOf<T>, ValueQuery>;
 
     impl<T: Config> Pallet<T> {
+        /// Getter for [`UnusedValue<T>`](UnusedValue).
+        pub fn unused_value() -> BalanceOf<T> {
```
[the remark this answers is not in the document]
Comment (playX18) [reply pallets/gear-bank/src/lib.rs:169]: What should happen when these types are used outside? We should add `pub use pallet::BlockNumber` or `pub use pallet::Bank`? Another approach is replace with path usage like `gear_bank::pallet::Bank`

[the remark this answers is not in the document]
Comment (breathx) [reply pallets/gear-bank/src/lib.rs:169]: For case somebody needs access to pallets storage, the storage should be `pub type`, or just keep such wrappers as `impl Pallet<T>` fns.

Atm Im mostly about inlining them when nobody external doesnt depend on it

[the remark this answers is not in the document]
Comment (playX18) [reply pallets/gear-bank/src/lib.rs]: Should be solved now

File: pallets/staking-rewards/src/lib.rs [hunk 2]
```diff
@@ -160,26 +160,22 @@ pub mod pallet {
 
     /// Target inflation (at ideal stake)
     #[pallet::storage]
-    #[pallet::getter(fn target_inflation)]
     pub(crate) type TargetInflation<T> = StorageValue<_, Perquintill, ValueQuery>;
 
     /// Ideal staking ratio
     #[pallet::storage]
-    #[pallet::getter(fn ideal_staking_ratio)]
```
Comment (ukint-vs) [inline pallets/staking-rewards/src/lib.rs:168]: All storage values from this pallet are used in frontends and provide useful info. There's no point in keeping it private.

**Review by ukint-vs — CHANGES_REQUESTED** [review]
Please explain the purpose of removing these getters. Because each getter can be easily accessed by clients without having to manually calculate the storage key.

**Review by ukint-vs — APPROVED** [review]
discussed

**Pull request merged**